app/geant-exporter/geant-exporter.cc

```diff
@@ -0,0 +1,221 @@
+         * crash Celeritas during the creation of its ParticleParams.
+         */
+        if (g4_particle_def->GetParticleName() == "geantino")
+            continue;
+
+        GeantParticle particle(g4_particle_def->GetParticleName(),
+                               g4_particle_def->GetPDGEncoding(),
+                               g4_particle_def->GetPDGMass(),
+                               g4_particle_def->GetPDGCharge(),
+                               g4_particle_def->GetPDGSpin(),
+                               g4_particle_def->GetPDGLifeTime(),
+                               g4_particle_def->GetPDGStable());
+
+        // Converting lifetime of unstable particles to seconds
+        // For stable particles lifetime = -1
+        if (!particle.is_stable)
+        {
+            particle.lifetime *= 1E-9;
+        }
+
+        std::string branch_name = g4_particle_def->GetParticleName();
+
+        // ROOT does not like special characters in its TBranch names
+        replace_characters(branch_name, "+", "Plus");
+        replace_characters(branch_name, "-", "Minus");
+
+        // Gamma is a ROOT function. A TBranch with this name confuses ROOT
+        if (branch_name == "gamma")
+        {
+            branch_name = "photon";
+        }
+
+        // Creating a new branch for a new particle
+        TBranch* particle_branch
+            = tree_particles.Branch(branch_name.c_str(), &particle);
+
+        particle_branch->Fill();
+        
+        // Writing the new branch in the same tree 
+        tree_particles.Write("", TObject::kOverwrite);
+
+        std::cout << "  Added " << branch_name << std::endl;
+    }
+
+    root_output.Write();
+    root_output.Close();
+
+    std::cout << root_output_filename << " done!" << std::endl;
+    std::cout << std::endl;
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Loops over particles and stores their physics tables
+ */
+void store_physics_tables(std::string root_output_filename,
+                          G4ParticleTable::G4PTblDicIterator* particle_iterator)
+{
+    GeantPhysicsTableParser table_parser;
+    table_parser.root_file_name(root_output_filename);
+
+    particle_iterator->reset();
+
+    while ((*particle_iterator)())
+    {
+        // Fetching particle and its data
+        G4ParticleDefinition* particle = particle_iterator->value();
+
+        // See comments in store_particles() on why we skip the geantino
+        if (particle->GetParticleName() == "geantino")
+            continue;
+
+        std::cout << "=============" << std::endl;
+        std::cout << particle->GetParticleName() << std::endl;
+        std::cout << "=============" << std::endl;
+
+        G4ProcessVector* process_list
+            = (particle->GetProcessManager())->GetProcessList();
+
+        // Looping over processes
+        for (std::size_t j = 0; j < process_list->size(); j++)
+        {
+            G4VProcess* process = (*process_list)[j];
```

Comment:
Is `G4ProcessVector` a standard container? This loop would be cleaner as
```
for (G4VProcess* process : *process_list)
{
```
since you don't need `j` anywhere else.